.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs/*
```

check the [common mistakes ](https://mate-academy.github.io/jv-program-common-mistakes/java-core/logger/logger)at the end of the task description before submitting your solution to review
```suggestion
logs/*

```